src/LayerSwitcher/LayerSwitcher.tsx

```diff
@@ -26,15 +24,19 @@ export interface OwnProps {
   /**
    * The layers to be available in the switcher.
    */
-  layers: ArrayTwoOrMore<OlLayerTile<OlTileSource> | OlLayerGroup>;
+  layers: OlLayerBase[];
```

Comment:
Is a layer group supported as input layer? If not we might want to set the type to `ol/layer/Layer`.

Reply:
LayerGroup is still supported.